### PR TITLE
Use new i18naddress API

### PR DESCRIPTION
**Pull request by artursmet**
Depends on https://github.com/mirumee/google-i18n-address/pull/16

### CODE REVIEW

File: saleor/checkout/views/shipping.py [hunk 1]
```diff
@@ -4,25 +4,35 @@
...
 from ...userprofile.models import Address
+from ...i18n_address_forms.forms import COUNTRY_FORMS
 
 
 def anonymous_user_shipping_address_view(request, checkout):
     data = request.POST or None
     shipping_address = checkout.shipping_address
+    country_code = request.country.code if request.country else 'US'
+
+    address_form_class = COUNTRY_FORMS[country_code]
```
Comment (patrys) [inline saleor/checkout/views/shipping.py]: What if GeoIP supports a country that we don't?

File: saleor/core/middleware.py [hunk 2]
```diff
@@ -33,10 +34,12 @@ def process_request(self, request):
 class CountryMiddleware(object):
 
     def process_request(self, request):
+        country_code = None
         if 'REMOTE_ADDR' in request.META:
-            request.country = get_country_by_ip(request.META['REMOTE_ADDR'])
-        else:
-            request.country = None
+            country_code = get_country_by_ip(request.META['REMOTE_ADDR'])
+        if settings.DEBUG and 'debug_country_code' in request.GET:
```
Comment (patrys) [inline saleor/core/middleware.py]: Shouldn't we instead allow country to be switched like we switch languages?

File: saleor/checkout/views/shipping.py (hunk 1 above)
Comment (patrys) [inline saleor/checkout/views/shipping.py]: How do I enter a valid shipping address for China if my GeoIP returns Poland?

**Comment by artursmet** [comment]
We need to add unsupported countries list to documentation as well.

**Pull request merged**